contracts/linear/src/validator_pool.rs

```diff
@@ -568,6 +570,7 @@ impl LiquidStakingContract {
             !validator.pending_release(),
             ERR_VALIDATOR_WITHDRAW_WHEN_LOCKED
         );
+        require!(validator.draining, ERR_NOT_IN_DRAINING);
```

Comment:
For assertion is good, the comment here is about the permission control of `drain_withdraw()`.

Would it be better to make `drain_withdraw()` callable by anyone? So it can be executed automatically by operator, rather than require manual invocation by manager account.

Reply:
yeah it can be public